babel.config.js

```diff
@@ -0,0 +1,57 @@
+module.exports = {
+    presets: [
+        [
+            '@babel/preset-env',
+            {
+                modules: false,
+            },
+        ],
+        '@babel/preset-react',
+        '@babel/preset-flow',
+    ],
+    plugins: [
+        '@babel/plugin-transform-flow-strip-types',
+        '@babel/plugin-transform-object-assign',
+        '@babel/plugin-proposal-class-properties',
+        '@babel/plugin-proposal-object-rest-spread',
+        [
+            'react-intl',
+            {
+                enforceDescriptions: true,
+                messagesDir: './i18n/json',
+            },
+        ],
+    ],
+    env: {
+        dev: {
+            plugins: ['flow-react-proptypes'],
+        },
+        npm: {
+            plugins: [
+                [
+                    'babel-plugin-transform-require-ignore',
```

Comment:
I think we should be able to remove this one completely from npm.
So that babel output has SCSS imports, which makes it unnecessary to import the CSS file.
(like BRUI)

But that can be another commit.

Reply:
I'd like to do that separately, since I believe it would be a breaking change. It would probably lead to duplicate stylesheets showing up for users who import the components, as they may already have the compiled stylesheet imported at the top level of their project.